test/jdk/java/util/Random/RandomTestBsi1999.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2020 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Shouldn't this be:
2012, 2021,

Reply:
The earlier dates were while under development. As far as I know, this is the first time they appear.